plugins/content-no-strings/content-no-strings.js

```diff
@@ -16,14 +16,10 @@ module.exports = stylelint.createPlugin(ruleName, (primaryOption) => {
       return;
     }
 
-    const allowedContentValues = [
-      "''",
-      '""',
-      'none',
-    ];
+    const nonEmptyString = /('[^']+')|("[^"]+")/;
```

Comment:
If you include the /('[^`'`]+')|("[^`"`]+")/; those characters to your regex you will also validate this:

<img width="256" alt="screen shot 2018-06-29 at 5 11 19 pm" src="https://user-images.githubusercontent.com/2091116/42115154-cfed07d4-7bbf-11e8-9af6-1bafa9ffb6ea.png">

Which is not super bad since I don't see an use case for that. But if you just remove that it will be able to catch that as an invalid value. Like this:

<img width="212" alt="screen shot 2018-06-29 at 5 11 03 pm" src="https://user-images.githubusercontent.com/2091116/42115181-e63bf2f2-7bbf-11e8-91c4-e49dcd2d86c0.png">

And as you can see it also works for the other cases:

<img width="258" alt="screen shot 2018-06-29 at 5 10 52 pm" src="https://user-images.githubusercontent.com/2091116/42115197-ee805606-7bbf-11e8-9f92-af30a47d6416.png">

Reply:
`content: '''` isn't valid CSS so I didn't bother to worry about such cases. Good to know, though!

Reply:
Our chat convinced me `[^]` is more robust than `[^']`. I think `.` is equivalent to `[^]`....